### DISCUSSION TITLE
Icon Request: fa-transmit fa-recieve

### DISCUSSION THREAD

**Discussion by brokentusk** [FortAwesome/Font-Awesome#21124] [Icon Requests]
This is something I seem to need every once and a while. Transmit and receive. Granted I spend a lot of time in the communications test industry, but there isn't an icon out there that fits when it comes to radio signals. The wifi bands works great, until you have to signify a particular direction for a signal. There's a kind of standard transmit icon, which is a pin with waves coming off it, but there isn't one for the other direction. Granted it has limited appeal, like the space-shuttle or code-fork icons, but there is a need for some kind of representation. 

![transmitreceive](https://cloud.githubusercontent.com/assets/22197007/19532783/d811af72-9602-11e6-9c8a-83edc4a7f8c9.png)

**Comment by Roumed** [comment]
+1

**Comment by ghost** [comment]
+1

**Comment by elmer-t** [comment]
Yes please!

**Comment by somethingSTRANGE** [comment]
+1

signal-send, signal-transmit, signal-broadcast
signal-receive, signal-listen

This would be used to indicate whether something is set to send (broadcast) or receive (listen for) a signal. Perhaps something like this:

![signal-send-and-receive](https://user-images.githubusercontent.com/6905832/50796165-8dc59f80-1285-11e9-9158-968cfae5abb1.png)


I suppose a third variation that repeats or boosts a signal would also be useful.

signal-repeat, signal-boost

![signal-repeat](https://user-images.githubusercontent.com/6905832/50796168-91592680-1285-11e9-8023-7c5f241159ba.png)

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by sensibleworld** [comment]
https://fontawesome.com/icons/tower-broadcast?f=classic&s=solid
https://fontawesome.com/icons/tower-receive?f=classic&s=solid
